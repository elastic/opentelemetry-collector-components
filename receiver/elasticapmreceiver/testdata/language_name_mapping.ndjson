{"metadata": {"service": {"name": "1234_service-12a3","node": {"configured_name": "node-123"},"version": "5.1.3","environment": "staging","language": {"name": "C#","version": "5"}, "agent": {"name": "elastic-dotnet", "activation_method": "some_activation_method","version": "3.14.0"}}, "system": {"hostname": "prod1.example.com","architecture": "x64","platform": "darwin", "container": {"id": "container-id"}}}}
{"transaction": { "id": "945254c567a5417e", "trace_id": "0123456789abcdef0123456789abcdef", "parent_id": "abcdefabcdef01234567", "type": "request", "duration": 32.592981, "timestamp": 1496170407154000, "span_count": { "started": 43 }, "otel": { "attributes" : { "elastic.profiler_stack_trace_ids" : ["foo", "bar", "foo"]}}}}
