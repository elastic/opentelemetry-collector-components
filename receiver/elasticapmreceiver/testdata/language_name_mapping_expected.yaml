resourceSpans:
  - resource:
      attributes:
        - key: service.name
          value:
            stringValue: 1234_service-12a3
        - key: service.version
          value:
            stringValue: 5.1.3
        - key: telemetry.sdk.language
          value:
            stringValue: dotnet
        - key: telemetry.sdk.name
          value:
            stringValue: ElasticAPM
        - key: deployment.environment.name
          value:
            stringValue: staging
        - key: agent.name
          value:
            stringValue: elastic-dotnet
        - key: agent.version
          value:
            stringValue: 3.14.0
        - key: service.language.name
          value:
            stringValue: C#
        - key: service.language.version
          value:
            stringValue: "5"
    scopeSpans:
      - scope: {}
        spans:
          - attributes:
              - key: timestamp.us
                value:
                  intValue: "1496170407154000000"
              - key: event.outcome
                value:
                  stringValue: success
              - key: processor.event
                value:
                  stringValue: transaction
              - key: transaction.id
                value:
                  stringValue: 945254c567a5417e
              - key: transaction.name
                value:
                  stringValue: ""
              - key: transaction.sampled
                value:
                  boolValue: true
              - key: transaction.root
                value:
                  boolValue: false
              - key: transaction.type
                value:
                  stringValue: request
              - key: transaction.result
                value:
                  stringValue: Success
              - key: transaction.duration.us
                value:
                  intValue: "32592981"
            endTimeUnixNano: "1496170407186592981"
            parentSpanId: abcdefabcdef0123
            spanId: 945254c567a5417e
            startTimeUnixNano: "1496170407154000000"
            status:
              code: 1
            traceId: 0123456789abcdef0123456789abcdef
